test/integration/patients/patient/patient-flow.js

```diff
@@ -389,6 +404,37 @@ context('patient flow page', function() {
       .find('.comment__item')
       .should('have.length', 3);
 
+    cy.sendWs({
+      category: 'AttachmentAdded',
+      resource: {
+        type: 'patient-actions',
+        id: testFlowAction.id,
+      },
+      payload: {
+        clinician: {
+          type: 'clinicians',
+          id: '95014c50-2401-4d95-83e1-4a39fb928c90',
+        },
+        file: {
+          type: 'files',
+          id: '70294032-539b-4dc0-93df-9e4ff2eee8c1',
+        },
+        attributes: {
+          path: `patients/${ testPatient.id }/HRA.pdf`,
+          view: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/view/HRA.pdf`,
+          download: `https://www.bucket_name.s3.amazonaws.com/patients/${ testPatient.id }/download/HRA.pdf`,
+        },
+      },
+    });
+
+    cy
+      .get('[data-attachments-files-region]')
+      .children()
+      .as('attachmentItems')
+      .should('have.length', 1)
+      .first()
+      .contains('HRA.pdf');
```

Comment:
Note to self: ensure that the attachment icon becomes visible for the action list item in this scenario.